frontend/src/components/HomePage/Data.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+const Data = ({ fakeData }) => {
+	// Different colors depending of the File size
+	const allFakeData = fakeData.map((data) => {
+		let color = "success";
```

Comment:
This can be a function on its own, defined outside of the component.

Reply:
Thanks for the suggestion, i will definitely change that.